src/session.js

```diff
@@ -475,6 +555,15 @@ ghostdriver.Session = function(desiredCapabilities) {
             }
         };
 
+        _decoratePromptBehavior(page);
+
+        // NOTE: The most common screen resolution used online is currently: 1366x768
+        // See http://gs.statcounter.com/#resolution-ww-monthly-201307-201312.
```

Comment:
I think you are right but what about putting this link (http://gs.statcounter.com/screen-resolution-stats/desktop/worldwide) and dating when this was last checked?